src/focus.service.ts

```diff
@@ -284,7 +284,14 @@ export class FocusService {
     this.focusRoot = newRootElem;
   }
 
-  public releaseFocus(scrollSpeed: number = Infinity) {
+  public releaseFocus(releaseElem?: HTMLElement, scrollSpeed: number = Infinity) {
+    if (releaseElem) {
+      if (releaseElem === this.focusRoot) {
+        this.releaseFocus(null, scrollSpeed);
+      }
+      return;
```

Comment:
The intent of this is to release the focus without worrying about if you trapped it in the first place or not. For instance, we can call it in ngOnDestroy hook.